views/page/reporting/choose-organisation.html

```diff
@@ -0,0 +1,103 @@
+{% from "../../component/reporting/macro.njk" import reporting with context %}
+{% from "radios/macro.njk" import govukRadios %}
+{% from "select/macro.njk" import govukSelect %}
+{% from "button/macro.njk" import govukButton %}
+{% from "error-summary/macro.njk" import govukErrorSummary %}
+
+{% extends "../../component/Page.njk" %}
+
+{% block pageTitle %}Reporting - Choose organisation - {{i18n['proposition_name'] }}{% endblock %}
+
+{% block content %}
+
+{% set typeAheadHtml %}
+
+<div class="govuk-form-group">
+  <label class="govuk-label" for="other-organisation-dropdown">
+    Organisation
+  </label>
+  <div class="autocomplete-wrapper autocomplete-wrapper-organisations ">
+
+    <select class="autocomplete__organisations" name="organisationId" id="parent">
+      <option value="" selected></option>
+      {% for organisation in organisationListForTypeAhead %}
+      <option value={{ organisation.id }}>{{ organisation.formattedName }}</option>
+      {% endfor %}
+    </select>
+  </div>
+</div>
+
+{% endset -%}
+
+{% if sessionFlash.errors %}
+    {{ govukErrorSummary({
+        titleText: "There is a problem",
+        errorList: [
+            {
+                text: sessionFlash.errors[0],
+                href: "#parent"
+            }
+        ]
+    }) }}
+{% endif %}
+
+<div class="govuk-grid row">
+  <div class="govuk-column-full">
+    <h1 class="govuk-heading-l">{{pageTitle}}</h1>
```

Comment:
This should be "Select organisation" in the tag. It's currently coming through empty which is failing accessibility checks.

Also needs the `<span class="govuk-caption-m">Reporting</span>` directly above it.

![image](https://github.com/Civil-Service-Human-Resources/lpg-management/assets/77386599/b24cc2d3-34fc-45d1-9bb0-86e433d3366e)